src/components/organisms/ActionsPane/ActionsPane.tsx

```diff
@@ -91,8 +91,33 @@ const ActionsPane = (props: {contentHeight: string}) => {
   const isActionsPaneFooterExpanded = useAppSelector(state => state.ui.isActionsPaneFooterExpanded);
   const kubeconfigPath = useAppSelector(state => state.config.kubeconfigPath);
   const [key, setKey] = useState('source');
+  const [isButtonShrinked, setButtonCollapsedState] = useState<boolean>();
```

Comment:
Should probably rename these to match each other

Reply:
Sure!

Reply:
Done